src/main/java/com/synopsys/integration/alert/provider/blackduck/actions/BlackDuckGlobalApiAction.java

```diff
@@ -77,8 +81,8 @@ public void afterDeleteAction(FieldModel fieldModel) {
         FieldValueModel fieldValueModel = keyToValues.get(ProviderDescriptor.KEY_PROVIDER_CONFIG_NAME);
         String blackDuckGlobalConfigName = fieldValueModel.getValue().orElse("");
 
-        Long configId = Long.parseLong(Objects.requireNonNullElse(fieldModel.getId(), "-1"));
-        providerLifecycleManager.unscheduleTasksForProviderConfig(blackDuckProvider, configId);
+        Long configId = Long.parseLong(fieldModel.getId());
```

Comment:
By removing the default "-1" aren't you opening this up to possible NPE's?

Reply:
There is no way for this config to be deleted without the id, so it will always be present or error out before it gets here.